app/src/main/java/com/codepath/apps/critterfinder/models/PetModel.java

```diff
@@ -35,6 +35,15 @@ public String getSex() {
         return sex;
     }
 
+    public String getSexFullName() {
+
+        switch (sex) {
+            case "M" : return "Male";
+            case "F" : return "Female";
+        }
+        return "";
+    }
```

Comment:
very cool! One suggestion - we end up showing hard coded strings that aren't localized by doing it this way. I created a string array of localized strings from the gender as part of search.

Check out our arrays.xml file:

```
    <string-array name="gender">
        <item>Male</item>
        <item>Female</item>
        <item>Both</item>
    </string-array>
```

To extract the localized string you could do something roughly like what we did in search:

String[] genderViewStrings = res.getStringArray(R.array.gender);
return genderViewStrings.get(SearchFilter.gender.ordinal());

And now you've got a locale friendly gender string! Although since our pet model doesn't use the same enum for gender you'll have to massage that part a tad.

I don't think iOS has an equivalent of a localized string array - that was something new that I thought was pretty cool about Android when wiring up gender strings in search.